### DISCUSSION TITLE
Access on MCU memory via USB / Explorer window

### DISCUSSION THREAD

**Discussion by Zelsof** [stm32duino/Arduino_Core_STM32#2149] [Q&A] [answered]
Hi,

I am struggeling with a question I didn't found any answer yet. 

I own an STM32 F446RE  developer board, and I noticed, when plugging it to my computer, that an explorer window opens with the files "DETAILS.TXT" and "MBED.HTM" in it.  SO i am wondering, if I can generate here small textfiles (like settings.txt), which can be later read by the mikrocontroller (like an laternative for an SD card). Is there any chance to have access to this memory?

Does anybody know something about it?

**Comment by fpistm** [comment]
It is linked to the STLink firmware.  Don't know if source is available publicly. You should ask on st community.

**Comment by Zelsof** [comment, marked as answer]
well it seems not to work. If anybody is interested, see https://community.st.com/t5/stm32-mcus-boards-and-hardware/stm32f429i-disc1-how-to-read-files-written-through-stlink-usb/td-p/399271

topic can be closed
